Gopkg.toml

```diff
@@ -26,8 +26,8 @@
 
 
 [[constraint]]
-  branch = "ss-master"
-  name = "github.com/shipstation/go-teamcity-sdk"
+  name = "github.com/cvbarros/go-teamcity-sdk"
```

Comment:
Why would you add the repo from cvbarros back? it does not work with our version of the provider anymore?